lib/v2/main.dart

```diff
@@ -86,14 +89,24 @@ class MainScreen extends StatelessWidget {
         switch (state.authStatus) {
           case AuthStatus.emptyAccount:
           case AuthStatus.recoveryMode:
-            return SeedsMaterialApp(
-              home: state.authStatus == AuthStatus.emptyAccount
-                  ? const OnboardingScreen()
-                  : SeedsMaterialApp(
-                      home: LoginScreen(),
-                    ),
-              navigatorKey: navigationService.onboardingNavigatorKey,
-              onGenerateRoute: navigationService.onGenerateRoute,
+            return BlocBuilder<DeeplinkBloc, DeeplinkState>(
+              builder: (context, deepLinkState) {
```

Comment:
why not just "state"?

Reply:
there is a state already declared higher up.

Reply:
But it is in its own closure that state. The BlocBuilder belongs to another bloc and another state, if you put the cursor on each state it will recognize what type it is.